### PR TITLE
New package: hatari-1.8.0

### PR DESCRIPTION
**Pull request by jarbowski**

### CODE REVIEW

**Comment by xtraeme** [comment]
This is completely wrong...

**Comment by jarbowski** [comment]
No es constructivo, caro dueño...

**Comment by xtraeme** [comment]
I'm so sorry but I'm tired of always answering the same stuff for some years already... I think the Manual.md file contains all required information, if you think something is missing, please enlighten me.

**Comment by jarbowski** [comment]
You're right. I'll face it by myself. Thank you for all the work done. This is an amazing distro. I think I won't come back to Arch.

File: srcpkgs/hatari/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'hatari'
+pkgname=hatari
+version=1.8.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="cmake pkg-config"
+makedepends="SDL-devel zlib-devel libpng-devel portaudio-devel"
+depends="SDL zlib libpng portaudio"
```
Comment (xtraeme) [inline srcpkgs/hatari/template]: This depends is unnecessary, remove it

File: srcpkgs/hatari/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'hatari'
+pkgname=hatari
+version=1.8.0
+revision=1
+build_style=cmake
+configure_args="--prefix=/usr"
```
Comment (xtraeme) [inline srcpkgs/hatari/template]: This line is unnecessary... remove it.

**Comment by xtraeme** [comment]
Also `update template` is not a proper commit log.

The file `CONTRIBUTING.md` has details for proper commit logs.

**Comment by xtraeme** [comment]
I won't pull this PR as is because you've made 12 commits and 11 with the same commit log.

Please rebase and submit a commit with a proper log message.

**Pull request closed without merging**